.github/workflows/e2e-test-with-domain.yaml

```diff
@@ -94,3 +96,17 @@ jobs:
         with:
           name: e2e-test-artifacts
           path: test/e2etest/artifacts/
+
+      - name: Send slack notification
+        if: always()
+        run: |
+          export GITHUB_EVENT_NUMBER="${{ github.event.number}}"
+          export JOB_STATUS="${{ job.status }}"
+          export PULL_REQUEST_URL="${{ github.event.pull_request._links.html.href }}"
+          export PULL_REQUEST_TITLE="${{ github.event.pull_request.title }}"
+          export SLACK_WEBHOOK_URL=${SLACK_WEBHOOK_URL_NOTIFICATIONS}
+          scripts/send_slack_notifications.sh
+          if [[ ${JOB_STATUS} == "failure" ]]; then
```

Comment:
Send message to alerts channel if the build fails (in addition to notifications channel). Each channel uses its own incoming webhook.